lib/identity/client.js

```diff
@@ -332,9 +336,14 @@ class Client extends PartialClient {
     )
     const groupLength = groupList.groups.length
     for (var index = 0; index < groupLength; index++) {
-      let sharedRecords = await this.storage.listRecordsSharedWithGroup(
-        groupList.groups[index].groupID
-      )
+      let sharedRecords = []
+      try {
+        sharedRecords = await this.storage.listRecordsSharedWithGroup(
+          groupList.groups[index].groupID
+        )
+      } catch (e) {
+        continue
```

Comment:
at the very least we should log this error, at the most we should collect them and return it as part of the response from this function